src/Sarif.Driver/Sdk/AnalyzeOptionsBase.cs

```diff
@@ -69,5 +69,12 @@ public class AnalyzeOptionsBase : IAnalyzeOptions
             Separator = ';',
             HelpText = "Path to plug-in that will be invoked against all targets in the analysis set.")]
         public IEnumerable<string> PluginFilePaths { get; set; }
+
+        [Option(
+            'i',
+            "invocation-properties",
+            Separator = ';',
+            HelpText = "Properties of the Invocation object to log. NOTE: StartTime and EndTime are always logged.")]
```

Comment:
when we support deterministic output, starttime and endtime will not appear. no call to action, just noting.